app/models/user.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const { Model } = require('sequelize');
+
+module.exports = (sequelize, DataTypes) => {
+  class user extends Model {
+    static associate(models) {
+      // define association here
+    }
+  }
+  user.init(
+    {
+      nombre: DataTypes.STRING,
+      apellido: DataTypes.STRING,
+      email: DataTypes.STRING,
+      contraseña: DataTypes.STRING
+    },
+    {
+      sequelize,
+      modelName: 'user'
+    }
+  );
+  return user;
+};
```

Comment:
It would be nice if you added the properties to each key of the model as in the migrations.
```
email: {
         allowNull: false,
         unique: true,
         type: Sequelize.STRING
       }
```